<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>367</width>
    <height>423</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Vaultary - password add</string>
  </property>
  <layout class="QGridLayout" name="gridLayout" rowstretch="1,1,1,1,1,1,1,1,1,1" columnstretch="1,0">
   <item row="4" column="0" rowspan="2" colspan="2">
    <layout class="QVBoxLayout" name="verticalLayout_3" stretch="0,1">
     <item>
      <widget class="QLabel" name="category_label">
       <property name="text">
        <string>Category</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="category_comboBox">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <item>
        <property name="text">
         <string/>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0" colspan="2">
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="password_label">
       <property name="text">
        <string>Password</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="password_lineEdit">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0" colspan="2">
    <layout class="QVBoxLayout" name="verticalLayout" stretch="1,1">
     <item>
      <widget class="QLabel" name="login_label">
       <property name="text">
        <string>Login</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="login_lineEdit">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="7" column="0" colspan="2">
    <layout class="QGridLayout" name="gridLayout_2" rowstretch="1,0,0" columnstretch="1,1,1,1,1,1">
     <item row="0" column="3" rowspan="3">
      <widget class="QCheckBox" name="special_checkBox">
       <property name="text">
        <string>%!@</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2" rowspan="3">
      <widget class="QCheckBox" name="numbers_checkBox">
       <property name="text">
        <string>0-9</string>
       </property>
      </widget>
     </item>
     <item row="1" column="4" colspan="2">
      <widget class="QSpinBox" name="length_spinBox">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>64</number>
       </property>
       <property name="value">
        <number>16</number>
       </property>
      </widget>
     </item>
     <item row="0" column="0" rowspan="3">
      <widget class="QCheckBox" name="capital_checkBox">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>A-Z</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="4" colspan="2">
      <widget class="QLabel" name="length_label">
       <property name="text">
        <string>Length</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1" rowspan="3">
      <widget class="QCheckBox" name="lowercase_checkBox">
       <property name="text">
        <string>a-z</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="3" column="0" colspan="2">
    <layout class="QVBoxLayout" name="verticalLayout_5">
     <item>
      <widget class="QLabel" name="URL_label">
       <property name="text">
        <string>URL</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="URL_lineEdit"/>
     </item>
    </layout>
   </item>
   <item row="9" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="generate_pushButton">
       <property name="text">
        <string>Generate password</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="clear_pushButton">
       <property name="text">
        <string>Clear</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="add_pushButton">
       <property name="text">
        <string>Add credential</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="0" colspan="2">
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <widget class="QLabel" name="service_label">
       <property name="text">
        <string>Service</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="service_lineEdit"/>
     </item>
    </layout>
   </item>
   <item row="6" column="0">
    <layout class="QVBoxLayout" name="verticalLayout_6">
     <item>
      <widget class="QLabel" name="notes_label">
       <property name="text">
        <string>Notes</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPlainTextEdit" name="notes_plainTextEdit"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
